### DISCUSSION TITLE
Version 17 via VCPKG/Conan

### DISCUSSION THREAD

**Discussion by LouChiSoft** [halide/Halide#8110] [Q&A] [answered]
Hi, I was wondering if there is a timeline for getting version 17 via VCPKG as well as the potential for a Conan package. We use Conan at work so it would be ideal if we could unify all our dependenices in one tool

**Comment by steven-johnson** [comment, marked as answer]
I'm not aware that anyone has every looked into packaging Halide with either. We'd be very happy to have it packaged with either or both, but we'd likely need a volunteer to implement and maintain it -- I don't think any of the current core contributors have the bandwidth to take this on.

**Reply by LouChiSoft** [reply]
You don't? So this package isn't maintained by the Halide team then?
![image](https://github.com/halide/Halide/assets/55987090/69856b9e-2062-4e9e-a997-b635b8a957b7)
I just assumed it was you guys since the GitHub link in the top right leads back to this repo

P.S. an image is the best I could do since the VCPKG website doesn't seem to have a shareable link to packages.

**Reply by steven-johnson** [reply]
I don't know who is responsible for that.

**Reply by LouChiSoft** [reply]
In that case I will stop using it, don't fancy using a package that isn't being maintained by the people who maintain the source. Thanks for confirming

**Reply by steven-johnson** [reply]
I will ask around and see if anyone knows who did this.

**Reply by abadams** [reply]
A variety of people update the Halide vcpkg package. Looks like it last happened due to an llvm version update. I think it's fairly common for third parties to do the vcpkg updates, though perhaps we should add it to our Halide release todo list:

https://github.com/microsoft/vcpkg/commits/master/ports/halide/vcpkg.json

**Reply by LouChiSoft** [reply]
An up to date vcpkg package would certainly be nice. Would still prefer Conan so that all our dependencies are under one tool, but since I can't provide that I won't expect it
